utils/s2n_init.c

```diff
@@ -89,12 +95,16 @@ static bool s2n_cleanup_atexit_impl(void)
     /* the configs need to be wiped before resetting the memory callbacks */
     s2n_wipe_static_configs();
 
-    return s2n_result_is_ok(s2n_cipher_suites_cleanup()) &&
+    bool cleaned_up = \
```

Comment:
Is there a reason you use "\\" here? I would think it's unnecessary.

Reply:
I blame python, or perl, or both.  Apologies.